en/docs/chapter_hashing/hash_map.md

```diff
@@ -1,30 +1,30 @@
 # Hash Table
 
-A "hash table", also known as a "hash map", achieves efficient element querying by establishing a mapping between keys and values. Specifically, when we input a `key` into the hash table, we can retrieve the corresponding `value` in $O(1)$ time.
+*Hash Table*, also known as *Hash Map*, achieves efficient querying by establishing a mapping between keys and values. Specifically, when we input a `key` into the hash table, we can get the corresponding `value` in $O(1)$ time complexity.
 
-As shown in the figure below, given $n$ students, each with two pieces of data: "name" and "student number". If we want to implement a query feature that returns the corresponding name when given a student number, we can use the hash table shown in the figure below.
+As figure shows below, given $n$ students, each with two attributes: "Name" and "Student ID". If we want to implement a query that returns the corresponding name of the given student ID, we can use the hash table shown in the figure below.
 
 ![Abstract representation of a hash table](hash_map.assets/hash_table_lookup.png)
 
-Apart from hash tables, arrays and linked lists can also be used to implement querying functions. Their efficiency is compared in the table below.
+Apart from hash tables, arrays and linked lists can also be used to query. The efficiency is compared in the table below.
 
-- **Adding Elements**: Simply add the element to the end of the array (or linked list), using $O(1)$ time.
-- **Querying Elements**: Since the array (or linked list) is unordered, it requires traversing all the elements, using $O(n)$ time.
-- **Deleting Elements**: First, locate the element, then delete it from the array (or linked list), using $O(n)$ time.
+- **Add**: Simply add the element to the end of the array (or linked list), with time complexity $O(1)$.
```

Comment:
I prefer `Adding Elements` here because:

- It is consistent with the Chinese version.
- `Add` is not semantically clear.
- Prefer nouns here.